front/public/mockServiceWorker.js

```diff
@@ -0,0 +1,322 @@
+/**
```

Comment:
what is this file?

Reply:
a useless file when I was testing msw, I'm gonna delete it